eo-maven-plugin/src/main/java/org/eolang/maven/name/OnReplaced.java

```diff
@@ -55,8 +55,16 @@ public final class OnReplaced implements ObjectName {
 
     /**
      * Delimiter between name and hash in EO object name.
+     * @todo #2394:30min Hide static constant DELIMITER.
+     *  We should hide static constant DELIMITER because it creates code duplication
+     *  in many places. For example in {@link OnReplaced#split()} and {@link OnVersioned#split()}
+     *  and {@link org.eolang.maven.Place#make(java.nio.file.Path, String)}.
+     *  Apparently we have to create a class which will parse raw string into two parts value and
+     *  optional version. Maybe this new class won't implement ObjectName interface.
+     *  Why static fields are bad you can read here:
+     *  - https://www.yegor256.com/2015/07/06/public-static-literals.html
      */
-    public static final String DELIMITER = "#";
+    public static final String DELIMITER = "|";
```

Comment:
@maxonfjvipon There is no one reason to make difference between `|` and `#` - it only confuses a reader. Why should we make it different then?

Reply:
@volodya-lombrozo ok, it was dictated by inability to save files with `|` on windows. Now we have `|` almost everywhere, but files are saved with `_`. Does not it confuse the reader?

Reply:
No, it doesn't, because you can clearly answer why it is so (the reason under it). Moreover this solution affects only the small part of the system comparing with the previous solution.